attestation-agent/lib/Cargo.toml

```diff
@@ -33,7 +33,7 @@ default = ["sample_kbc", "rust-crypto", "coco_as"]
 kbs = ["kbs_protocol/background_check"]
 coco_as = ["reqwest"]
 
-cc_kbc = ["kbc/cc_kbc", "kbs_protocol/background_check", "kbs"]
+cc_kbc = ["kbc/cc_kbc", "kbs_protocol/background_check", "kbs", "coco_as"]
```

Comment:
This is interesting that `cc_kbc` is an old name to specify the kbc. But now kbc implementations are moved to CDH, thus only attester/platform should be configured for AA.

In future, we might also need to delete all `*-kbc` features and instead only leave `*-attester` and different kinds of token features.

But currently, it is ok to me.